test/_test-track.jl

```diff
@@ -24,19 +26,24 @@ cliparams = [
     params_path,
 ]
 
-obj = deserialize(data_path)
+obj = deserialize(joinpath(data_path, "tracker_test_data.dat"))
+passtimes = deserialize(joinpath(data_path, "passtimes.dat"))
+
+# Keep floes with area >= 350 pixels
```

Comment:
We should mention somewhere that 350 seems to be a minimum size, maybe in the docstrings for `feature-extraction.jl` and update the `minfloearea` default to 350

Reply:
That probably belongs in a separate PR? That is parameter of feature extraction.